snowflake/plugin.go

```diff
@@ -19,24 +19,29 @@ const pluginName = "steampipe-plugin-snowflake"
 func Plugin(ctx context.Context) *plugin.Plugin {
 	p := &plugin.Plugin{
 		Name:             pluginName,
-		DefaultTransform: transform.FromGo().Transform(valueFromNullable),
+		DefaultTransform: transform.FromCamel().Transform(valueFromNullable),
```

Comment:
Why the switch from `FromGo` to `FromCamel`? What do most tables need for transform?